File: Python/EX1.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        print(f'Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}')

# Instanciando três objetos da classe Carro
carro1 = Carro("Toyota", "Yaris", 2020)
carro2 = Carro("Honda", "Accord", 2019)
carro3 = Carro("Ford", "Edge", 2018)

# Exibindo os detalhes de cada carro
carro1.exibir_detalhes()
carro2.exibir_detalhes()
carro3.exibir_detalhes()

File: Java/EX1.java
class Carro {
    private String marca;
    private String modelo;
    private int ano;

    // Construtor
    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    // Método para exibir os detalhes
    public void exibirDetalhes() {
        System.out.println("Marca: " + this.marca + ", Modelo: " + this.modelo + ", Ano: " + this.ano);
    }

    // Método principal
    public static void main(String[] args) {
        // Instanciando três objetos da classe Carro
        Carro carro1 = new Carro("Toyota", "Yaris", 2020);
        Carro carro2 = new Carro("Honda", "Accord", 2019);
        Carro carro3 = new Carro("Ford", "Edge", 2018);

        // Exibindo os detalhes de cada carro
        carro1.exibirDetalhes();
        carro2.exibirDetalhes();
        carro3.exibirDetalhes();
    }
}
